### ISSUE TITLE
AG WG Review: 1.4.6 Contrast (Enhanced) (Level AAA) WCAG2ICT Guidance

### ISSUE THREAD

**Issue by pday1** [w3c/wcag2ict#892]
## Summary
Review of proposed WCAG2ICT guidance for the Level AAA WCAG SC: 1.4.6 Contrast (Enhanced), as incorporated into the Editor's draft. 

## Maturity Level
Refining

## Action Needed
* Review content (See Content for Review section below for details.)
* Use the thumbs up and thumbs down icons to indicate agreement or disagreement with various comments. Do not add comments to simply support other comments. 
* Note any suggested edits for improvement and/or reasoning either in a comment on this issue or by creating a PR on the WCAG2ICT document.  
* Level A and AA SCs can be found in the [comments-by-guideline-and-success-criteria.md](https://github.com/w3c/wcag2ict/blob/main/comments-by-guideline-and-success-criterion.md) file.
* Level AAA SCs can be found in the [comments-on-level-AAA-success-criteria.md](https://github.com/w3c/wcag2ict/blob/main/comments-on-level-AAA-success-criteria.md)
----

## Background information 

Once finalized, this new WCAG2ICT Note will build on the previous version by also covering WCAG 2.2 level AAA success criteria (currently only level A and AA criteria are covered.  This is important as different jurisdictions are using different versions of WCAG for the accessibility of information and communication technology. 

## Content for Review

* Section [Applying SC 1.4.6 Contrast (Enhanced) to non-web documents and non-web software](https://w3c.github.io/wcag2ict/#applying-sc-1-4-6-contrast-enhanced-to-non-web-documents-and-non-web-software).  

## Reference Material
In case you need them, links to various resources:

* [1.4.6 Contrast (Enhanced)](https://www.w3.org/TR/WCAG22/#contrast-enhanced) SC text in WCAG 2.2
* [Intent section of Understanding 1.4.6 Contrast (Enhanced)](https://www.w3.org/WAI/WCAG22/Understanding/contrast-enhanced#intent)

**Comment by maryjom** [comment]
Note 1 is all about hardware and not about non-web documents or non-web software - so you cannot declare those as being out of scope. 

Notes 1, 2, and 3 all seem to be circling the same topic. I suggest that these are combined in some manner. That the content author may or may not have full control over the contrast of content displayed on certain hardware using some types of displays. As an example you might document that content displayed on reflective displays may not give the appearance of good contrast, depending on the lighting conditions- and the content author may not have the ability to make sufficient contrast adjustments to support the contrast level required by this SC. You might suggest that content be tested on emissive displays and saying that content displayed on reflective displays may not be able to satisfy this success criterion. 

Note 4 is not needed. Earlier in WCAG2ICT (the Excluded from Scope section) it is already stated that this document does not apply to hardware:
>WCAG2ICT does not comment on hardware aspects of products because the basic constructs on which WCAG 2 is built do not apply to hardware.

**Comment by pday1** [comment]
Current content in editor's draft:

Applying SC 1.4.6 Contrast (Enhanced) to non-web documents and non-web software

> This applies directly as written, and as described in [Intent from Understanding Success Criterion 1.4.6](https://www.w3.org/WAI/WCAG22/Understanding/contrast-enhanced#intent).
> 
> NOTE 1 (ADDED)
> This success criterion only applies to ICT with emissive displays. Reflective displays are out of scope.
> NOTE 2 (ADDED)
> It may be more difficult to meet this success criterion when the author of the non-web software or non-web document can only modify the appearance of content within a limited contrast range determined by hardware or platform.
> NOTE 3 (ADDED)
> When the contrast of the content is determined by the hardware and not modifiable by the author of the non-web software or non-web document, it may not be possible to meet this success criterion.
> NOTE 4 (ADDED)
> Contrast requirements for hardware are out of scope for WCAG2ICT (and this success criterion).
> NOTE 5 (ADDED)
> When the color contrast ratio cannot be programmatically measured due to system limitations (e.g. lockdown), precise quantifiable testing of color contrast cannot be performed by a third party. In such cases, the author of the non-web software or non-web document would need to confirm that the color combinations used meet the contrast requirement.
> NOTE 6 (ADDED)
> Photographs (e.g., of a hardware display) are not sufficient for testing that content meets this success criterion. This is because the quality of the lighting, camera, and physical aspects of the hardware display can dramatically affect the ability to capture the content for testing purposes.
